op-e2e/testdata/challenge_test_data.go

```diff
@@ -13,24 +14,61 @@ import (
 	"github.com/kroma-network/kroma/kroma-bindings/bindings"
 )
 
+type ProofType string
+
+const (
+	ZkVMType         ProofType = "zkVM"
+	ZkEVMType        ProofType = "zkEVM"
+	DefaultProofType           = ZkEVMType
+)
```

Comment:
How about using enum?  (This is just an opinion and you don't have to change it.)

```suggestion
type ProofType int

const (
	ZkVMType  ProofType = iota
	ZkEVMType ProofType
)
const  DefaultProofType = ZkEVMType
```

Reply:
I referred to DataAvailabilityType